src/AzureAppConfigurationImpl.ts

```diff
@@ -79,6 +127,75 @@ export class AzureAppConfigurationImpl extends Map<string, unknown> implements A
         }
     }
 
+    /**
+     * Refresh the configuration store.
+     */
+    public async refresh(): Promise<void> {
+        if (!this.#refreshEnabled) {
+            return Promise.resolve();
+        }
+
+        // if still within refresh interval/backoff, return
+        if (this.#refreshTimer.canRefresh()) {

```

Comment:
```suggestion
        if (!this.#refreshTimer.canRefresh()) {

```

Reply:
by bad. wondering why the test case didn't cover it. Let me check.

Reply:
Got it. I didn't change the impl either.
```
    public canRefresh(): boolean {
        return Date.now() < this._backoffEnd;  // should be >=
    }
```

Reply:
fixed in 0e347be